step-capstone/src/components/Sidebars/TimeLine.js

```diff
@@ -37,6 +37,10 @@ export default class TimeLine extends React.Component {
   componentDidUpdate(prevProps) {
     if (prevProps.displayDate.toDateString() !== this.props.displayDate.toDateString) {
```

Comment:
should it be:
if (prevProps.displayDate.toDateString() !== this.props.displayDate.toDateString())?